apps/food-server/src/datasources/auth0-api.ts

```diff
@@ -0,0 +1,42 @@
+import { DataSource } from 'apollo-datasource';
+import axios, { AxiosRequestConfig } from 'axios';
+
+export class Auth0API extends DataSource {
```

Comment:
Don't have to do this now, but since we are duplicating this code it would make sense to move it to an `auth` library at some point (along with any other duplicate auth code)